modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -392,6 +358,11 @@ public CouchbaseContainer withIndexMemoryQuota(String indexMemoryQuota) {
         return self();
     }
 
+    public CouchbaseContainer withClusterUsername(String clusterUsername) {
```

Comment:
I wonder if this should be withAdminUsername / withAdminPassword ? to distinguish it from real RBAC users

Reply:
I like the word cluster for this context. it allows me to manage the whole thing.

Reply:
what about ClusterAdmin ? makes both the scope and the role clear.